python-package/xgboost/sklearn.py

```diff
@@ -938,8 +938,11 @@ def fit(
                 Use `early_stopping_rounds` in :py:meth:`__init__` or
                 :py:meth:`set_params` instead.
         verbose :
-            If `verbose` and an evaluation set is used, writes the evaluation metric
-            measured on the validation set to stderr.
+            If `verbose` is True and an evaluation set is used, the evaluation metric
+            measured on the validation set is printed to stderr at each boosting stage.
```

Comment:
It's printed to stdout.

Reply:
thx, I didn't verify this myself. Updated now for both occurrences.